plugins/modules/at.py

```diff
@@ -151,6 +157,7 @@ def main():
     )
 
     at_cmd = module.get_bin_path('at', True)
+    atrm_cmd = module.get_bin_path('atrm', True)
```

Comment:
Does `atrm` already exist on newer CentOS and RHEL machines?

Reply:
Hi thanks for the review!
Yeah, I have confirmed the following version has atrm command:
* RHEL6.10
* RHEL7.9
* RHEL8.4
* FreeBSD 13.0-STABLE
* NetBSD  9.1
* OpenBSD 6.9
* Solaris 11.4.0.15.0